File: tool/src/org/antlr/v4/parse/v3TreeGrammarException.java
/*
 * Copyright (c) 2012 The ANTLR Project. All rights reserved.
 * Use of this file is governed by the BSD-3-Clause license that
 * can be found in the LICENSE.txt file in the project root.
 */

package org.antlr.v4.parse;

import org.antlr.runtime.Token;
import org.antlr.v4.runtime.misc.ParseCancellationException;

public class v3TreeGrammarException extends ParseCancellationException {
	private static final long serialVersionUID = -8383611621498312969L;

	public Token location;

	public v3TreeGrammarException(Token location) {
		this.location = location;
	}
}

File: runtime/CSharp/Antlr4.Tool/Parse/v3TreeGrammarException.cs
// Copyright (c) Terence Parr, Sam Harwell. All Rights Reserved.
// Licensed under the BSD License. See LICENSE.txt in the project root for license information.

namespace Antlr4.Parse
{
    using IToken = Antlr.Runtime.IToken;
    using ParseCanceledException = Antlr4.Runtime.Misc.ParseCanceledException;

    public class v3TreeGrammarException : ParseCanceledException
    {
        public IToken location;

        public v3TreeGrammarException(IToken location)
        {
            this.location = location;
        }
    }
}
